harness/determined/pytorch/_callback.py

```diff
@@ -88,13 +88,19 @@ def on_checkpoint_save_start(self, checkpoint: Dict[str, Any]) -> None:
 
     def on_checkpoint_end(self, checkpoint_dir: str) -> None:
```

Comment:
optional, but what do you think about renaming this to `on_checkpoint_write_end()`, and leave a deprecated alias for it called `on_checkpoint_end()`?

Reply:
Agree entirely. I expanded the `on_checkpoint_end` doc string because I thought the method name would be extra confusing after the addition of `on_checkpoint_upload_end`